metal/config.go

```diff
@@ -59,5 +64,13 @@ func (c *Config) Client() *packngo.Client {
 	retryClient.RetryWaitMax = c.MaxRetryWait
 	retryClient.CheckRetry = packngo.RetryPolicy
 	httpClient := retryClient.StandardClient()
```

Comment:
can we rename the variable httpClient to "standardClient" so that it's doesn't clash with package "github.com/hashicorp/terraform-plugin-sdk/httpclient", it's a bit confusing to read

Reply:
logging this as https://github.com/equinix/terraform-provider-metal/issues/39 and merging